setup.py

```diff
@@ -103,7 +103,7 @@ def run(self):
     author_email=EMAIL,
     python_requires=REQUIRES_PYTHON,
     url=URL,
-    packages=find_packages(exclude=["tests", "casbin_mongoengine_adapter"]),
+    packages=find_packages(),
```

Comment:
I'm wondering, should we use:
```
exclude=["tests"]
```
rather than remove all?

Reply:
@Zxilly

Reply:
`tests` folder didn't have file `__init__.py`, so it won't be regarded as a module or package.

Reply:
I'm newbiew in python.
Thank your answer.